apps/dialtone-documentation/docs/.vuepress/baseComponents/SvgLoader.vue

```diff
@@ -16,6 +16,13 @@ const props = defineProps({
     type: String,
     required: true,
   },
+  /**
+   * Class attribute to be added to the component.
+   */
+  class: {
+    type: String,
```

Comment:
For classes we use `type: [String, Array, Object],` since all are valid ways to set classes in vue.